.golangci.yml

```diff
@@ -62,10 +62,9 @@ linters-settings:
       - Prefix(github.com/karmada-io/karmada)
   revive:
     rules:
-      # Disable if-return as it is too strict and not always useful.
-      # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#if-return
-      - name: if-return
-        disabled: true
```

Comment:
This `if-return` rule is on or off now?

Reply:
In the newest version, it's disabled by default: https://github.com/golangci/golangci-lint/pull/4087/files#diff-a133b3199860b0c64f3827116b9a35c9154cd171062ec70c2c94591788f015a0